packages/x-data-grid/src/components/cell/GridActionsCell.tsx

```diff
@@ -204,6 +204,7 @@ function GridActionsCell(props: GridActionsCellProps) {
           touchRippleRef: handleTouchRippleRef(index),
           onClick: handleButtonClick(index, button.props.onClick),
           tabIndex: focusedButtonIndex === index ? tabIndex : -1,
+          role: numberOfButtons === 1 ? 'button' : 'menuitem',
```

Comment:
Why would this be a "menuitem" since it is a simple icon button and it has nothing to do with a "menu"?